index.js

```diff
@@ -161,6 +167,20 @@ const expressSwagger = (express, callback) => {
   })
 }
 
+const trackRequests = (express) => {
+    express.use((req, res, next) => {
+        const requestId = req.headers['x-request-id'];
+        log.info('creating namespace for ', req.method, req.url, requestId);
```

Comment:
Also I think we already logging method and path somewhere above, but may be wrong.